templates/default/node/nodeeditbox.html

```diff
@@ -158,10 +158,27 @@
 										<input type="hidden" name="nodeedit[location_street]" value="{$nodeinfo.location_street}">
 										<input type="hidden" name="nodeedit[location_house]" value="{$nodeinfo.location_house}">
 										<input type="hidden" name="nodeedit[location_flat]" value="{$nodeinfo.location_flat}">
-										<input type="checkbox" name="nodeedit[teryt]" id="teryt" onchange="check_teryt('location')"{if $nodeinfo.teryt} checked{/if}><label for="teryt">{trans("TERRIT-DB")}</label>
+										<input type="checkbox" name="nodeedit[teryt]" id="teryt" onchange="check_teryt('location')"{if $nodeinfo.teryt} checked{/if}> <label for="teryt">{trans("TERRIT-DB")}</label>
 									</td>
 								</tr>
 
+									<tr>
```

Comment:
W nodeedit masz przypisywanie w oddzielnej zakładce. W nodeadd nie ma zakładek, więc ktoś dorobił przypisanie do jednej grupy komputerów, a lepiej byłoby to przerobić, żeby do wielu grup można było od razu przypisać (podpowiedź: poprzez zmianę klasy dla select-a na **lms-ui-multiselect** (oczywiście w kodzie php trzeba to też uwzględnić).